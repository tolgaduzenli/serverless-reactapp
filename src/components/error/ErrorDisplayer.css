.error-container{
    display: flex;
    justify-content: center;
    align-items: center;
    top: 50px;
    background-color: #ef5350;
}