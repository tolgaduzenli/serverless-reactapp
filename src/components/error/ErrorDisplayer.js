import React from 'react'
import './ErrorDisplayer.css'

const ErrorDisplayer = () => {
    return <div className="error-container">
        <div>
            <p>Error has been occurred.Please try again later</p>
        </div>
    </div>
}

export default ErrorDisplayer