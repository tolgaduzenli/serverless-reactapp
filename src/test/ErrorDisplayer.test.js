import React from 'react'
import enzyme from 'enzyme'
import ErrorDisplayer from '../components/error/ErrorDisplayer'

describe('<ErrorDisplayer />', () => {
    let wrapper
    beforeEach(() => {
        wrapper = enzyme.shallow(
            <ErrorDisplayer
            />
        );
    });

    it('should render a <div />', () => {
        expect(wrapper.find('div').length).toEqual(2);
    });

    it('should render correctly', () => expect(wrapper).toMatchSnapshot());

})