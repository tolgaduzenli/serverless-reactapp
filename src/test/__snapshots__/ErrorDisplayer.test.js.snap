// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorDisplayer /> should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ErrorDisplayer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div>
        <p>
          Error has been occurred.Please try again later
        </p>
      </div>,
      "className": "error-container",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <p>
          Error has been occurred.Please try again later
        </p>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Error has been occurred.Please try again later",
        },
        "ref": null,
        "rendered": "Error has been occurred.Please try again later",
        "type": "p",
      },
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div>
          <p>
            Error has been occurred.Please try again later
          </p>
        </div>,
        "className": "error-container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <p>
            Error has been occurred.Please try again later
          </p>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Error has been occurred.Please try again later",
          },
          "ref": null,
          "rendered": "Error has been occurred.Please try again later",
          "type": "p",
        },
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
